core/src/main/java/io/questdb/cairo/pool/AbstractMultiTenantPool.java

```diff
@@ -400,8 +407,12 @@ protected boolean releaseAll(long deadline) {
                             if (deadline == Long.MAX_VALUE) {
                                 r.goodbye();
                                 Unsafe.arrayPutOrdered(e.allocations, i, UNALLOCATED);
-                                LOG.info().$("shutting down, table is left behind [table=").$(r.getTableToken())
-                                        .I$();
+                                var rec = LOG.info().$("shutting down, table is left behind [table=").$(r.getTableToken()).$(']');
+                                var supervisor = r.getSupervisor();
+                                if (supervisor instanceof TracingResourcePoolSupervisor<T>) {
+                                    ((TracingResourcePoolSupervisor<T>) supervisor).printResourceInfo(rec.$(": "), r);
+                                }
+                                rec.$();
```

Comment:
we have to make try-finally block out of this, the $() is a must, even if "print" fails